modules/api/src/main/java/org/apache/ignite/schema/definition/builder/PrimaryKeyDefinitionBuilder.java

```diff
@@ -39,6 +48,14 @@
      */
     PrimaryKeyDefinitionBuilder withColumns(String... columns);
 
+    /**
+     * Sets primary key columns.
+     *
+     * @param columns Indexed columns.
+     * @return {@code this} for chaining.
+     */
+    PrimaryKeyDefinitionBuilder withColumns(Set<String> columns);
```

Comment:
Are we going to build an index on primary key implicitly?
Undefined column order will restrict us here.